packages/css/README.md

```diff
@@ -49,3 +49,23 @@ To use the compact mode, import the compact css **after** theme css, like so:
 import "@amsterdam/design-system-tokens/dist/index.css";
 import "@amsterdam/design-system-tokens/dist/compact.css";
 ```
+
+## Document styles
+
+Some elements of your document are out of reach for the Design System components.
+However, their styles can influence the appearance of the components.
+Apply the following to the document elements:
+
+The font-size must not be changed – e.g. through `html { font-size: 62.5% }` – because the typography system expects a rem to be 16 pixels.
+
+The `ams-body` class must be added to the `<body>` element – it prepares the base layout and sets a background colour.
+
+After the `theme-ams` class has been set, e.g. on the `<body>` as well, bold text will use our extra bold font weight.
+
+## Common patterns
+
+We use Sass mixins to extract common patterns, especially if they need more than 1 declaration.
+We collect reset styles in mixins as well.
+Both the name of the mixins and their documentation help explain the approach.
+Mixins sit at the end of the declation block to ensure their effect.
+The general ‘reset’ mixin is the very last.
```

Comment:
Isn't this more of a code convention thing? It's not really relevant for our consumers, right?

Reply:
Correct, I’ve moved the section.